README.md

```diff
@@ -285,6 +282,112 @@ func main() {
 }
 ```
 
+To use SQL to query a tarantool instance, call `Execute`. 
+
+Pay attention that with different types of queries (DDL, DQL, DML etc.) 
+some fields of the response structure (`MetaData` and `InfoAutoincrementIds` in `SQLInfo`) may be nil.
+
+See the [protocol](https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#responses-for-sql) explanation for details.
+
+```go
```

Comment:
Please don't put example to the README. Put it to tests instead. Other examples I'll move in scope of #123.

See [Testable Examples in Go](https://go.dev/blog/examples) and https://pkg.go.dev/testing#hdr-Examples and existed examples in https://github.com/tarantool/go-tarantool/blob/master/example_test.go

Reply:
Fixed.